setup.py

```diff
@@ -629,6 +637,49 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
 
 # ----------------------------------------------------------------------
 
+# ----------------------------------------------------------------------
+# rdata
+
+rdata_srcs = [
+    "pandas/io/rdata/_rdata.pyx",
+    "pandas/_libs/src/librdata/rdata_parser.c",
+    "pandas/_libs/src/librdata/rdata_read.c",
+    "pandas/_libs/src/librdata/rdata_write.c",
+    "pandas/_libs/src/librdata/rdata_io_unistd.c",
+    "pandas/_libs/src/librdata/rdata_error.c",
+    "pandas/_libs/src/librdata/rdata_bits.c",
+    "pandas/_libs/src/librdata/CKHashTable.c",
+]
+
+if is_platform_windows():
+    rdata_srcs.append("pandas/_libs/src/librdata/win_iconv.c")
+
+include = []
+libs_dir = []
+libs = []
+if is_platform_mac():
+    os.environ["DYLD_LIBRARY_PATH"] = ""
+    include = ["/usr/include"]
```

Comment:
Why not handle the mac includes here rathe than up top?

Reply:
Didn't want to interfere with above extensions especially with modification of environment variable in session.

Reply:
But this include here is _only_ used by rdata_ext, so you can customize it for rdata_ext as needed rather than injecting the customization above.

Reply:
I think you should devine extra includes for this module in the special block if OSX that occurs earlier. You can then add these includes in the usual place in the dictionary and avoid all of the special casing I think.